src/main/java/org/wildfly/security/keystore/AtomicLoadKeyStoreSpi.java

```diff
@@ -46,6 +48,8 @@
 
     @Override
     public void engineLoad(InputStream stream, char[] password) throws IOException, NoSuchAlgorithmException, CertificateException {
+
+        ElytronMessages.tls.tracef("AtomicLoadKeyStore loading:  stream = %s,  password = %b", stream, password);
```

Comment:
Wouldn't it be better to pass "password != null" as tracef's argument? If someone changes the string and accidentally changes %b to %s, we have password logged.